go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
 	github.com/pelletier/go-toml v1.8.0 // indirect
+	github.com/pkg/errors v0.9.1
```

Comment:
huh do you know why this was added?

Reply:
I'm honestly not sure; I ran `go mod tidy` and I don't quite know why this got picked up. 

...

On further investigation it's added in the `task` pkg; looks like it [got auto-added by goimports](https://github.com/aws/copilot-cli/commit/894883886846b3014a2c175842dec6820dc75d6e) in #1145 when just "errors" would have worked.

Reply:
ooh @Louise15926 can you take a look at this, when working on the "task" pkg  PR can we remove the dependency from the external `errors` and use "import errors" instead?